### ISSUE TITLE
Bootstrap 4?

### ISSUE THREAD

**Issue by eduardomart** [eduardinni/bootlint-online#25]
Bootlint is incredibly useful, but I am not sure it's currently compatible with BS4.
Any info would be appreciated.

**Comment by coliff** [comment]
Correct, Bootlint is not currently compatible with Bootstrap 4, but I'm hoping that support is added at some point. There's an open PR with a lot of good work on adding BS4 support here:
https://github.com/twbs/bootlint/pulls
